tests/integration/test_api.py

```diff
@@ -54,3 +54,27 @@ def test_remove_snap(test_snap):
     response = wait_for(snap_http.remove)("test-snap")
     assert response.status_code == 202
     assert is_snap_installed("test-snap") is False
+
+
+def test_sideload_snap(local_test_snap_path):
+    """Test sideloading a snap."""
+    assert is_snap_installed("test-snap") is False
+
+    response = wait_for(snap_http.sideload)(
+        file_paths=[local_test_snap_path],
+        devmode=True,
```

Comment:
up to you if you think we need further integration tests with any of the other `sideload` boolean flags.